apps/desktop/src/lib/forge/github/githubPrMonitor.ts

```diff
@@ -42,6 +43,17 @@ export class GitHubPrMonitor implements ForgePrMonitor {
 		await this.fetch();
 	}
 
+	async pollForMergeableState(attempts?: number): Promise<void> {
+		const actualAttempts = attempts ?? 0;
+		if (actualAttempts >= MAX_POLL_ATTEMPTS) return;
+		const currentMergeableState = get(this.mergeableState);
```

Comment:
By direct do you mean something like a stand-alone function that just polls for the state?
If so I guess that we can do that, but we still need to set the information to a store so that the mergeable state is propagated

I guess another way would be to have the fetch function retry until there is data (as `getPrWithRetries` already does) and until `mergeableState` is not undefined